src/dynatrace/azext_dynatrace/aaz/latest/dynatrace/monitor/single_sign_on_configurations/_create.py

```diff
@@ -0,0 +1,303 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "dynatrace monitor sso-config create",
+)
+class Create(AAZCommand):
+    """Create a dynatrace single-sign-on resource
+
+    :example: Create a sso-config
+        az dynatrace monitor sso-config create -g rg --monitor-name monitor -n default --aad-domains [\'mpliftrdt20210811outlook.onmicrosoft.com\'] --single-sign-on-url "https://www.dynatrace.io"
```

Comment:
Please use double quote to wrap shorthand syntax.
```suggestion
        az dynatrace monitor sso-config create -g rg --monitor-name monitor -n default --aad-domains "['mpliftrdt20210811outlook.onmicrosoft.com']" --single-sign-on-url "https://www.dynatrace.io"
```